.gitignore

```diff
@@ -139,3 +139,8 @@ libsrc/*.zip
 # NuGet v3's project.json files produces more ignorable files
 *.nuget.props
 *.nuget.targets
+
+\.DS_Store
```

Comment:
Could you please add comments to these new entries? Also, there's already "XCode" section above, if DS_Store is relevant please move this item to same group to keep things organized.

Reply:
Fair point. Added comments. ds_store is an os level file like thumbs.db and desktop.ini, so doesn't quite fall under xcode.